src/app/lib/transaction.ts

```diff
@@ -79,6 +82,73 @@ export class OasisTransaction {
     return tw
   }
 
+  public static async buildStakingAllowTransfer(
```

Comment:
if not Allow, it should rather be AllowWithdraw

Reply:
oh I see the rest of the codebase uses a broader meaning of "transfer." carry on then :upside_down_face: